chronicles/textlineserializer.nim

```diff
@@ -77,15 +95,14 @@ proc writeArray*[T](w: var TextLineWriter, elements: openarray[T]) =
       w.stream.writeText ", "
   w.stream.writeText ']'
 
-# TODO: is this meant to be for key/value lists?
 proc writeIterable*(w: var TextLineWriter, collection: auto) =
-  w.stream.writeText '['
```

Comment:
An `iterable` is a list, so using `[ ]` seems more appropriate to me.

Reply:
Done.